ui-frontend/packages/catalog-ui-search/src/main/webapp/lib/cesium-drawhelper/DrawHelper.ts

```diff
@@ -556,22 +595,90 @@ const DrawHelper = (function () {
         width: this.width < 1 ? 1 : this.width,
         vertexFormat: Cesium.EllipsoidSurfaceAppearance.VERTEX_FORMAT,
         ellipsoid: this.ellipsoid,
+        followSurface: false,
+      })
+    }
+    _.prototype.getOutlineGeometry = function () {
+      if (
+        !Cesium.defined(this.positions) ||
+        this.positions.length < 2 ||
+        !this.buffer
+      ) {
+        return
+      }
+      const coordinates = this.positions.map((pos: Cesium.Cartesian3) => {
+        const cartographic = Cesium.Cartographic.fromCartesian(pos)
+        return [
+          Cesium.Math.toDegrees(cartographic.longitude),
+          Cesium.Math.toDegrees(cartographic.latitude),
+          cartographic.altitude,
+        ]
+      })
+      const turfLine = Turf.lineString(coordinates)
+      utility.adjustGeoCoords(turfLine)
+      const bufferedLine = Turf.buffer(turfLine, Math.max(this.buffer, 1), {
+        units: 'meters',
+      })
+      if (!bufferedLine) {
+        return
+      }
+      // need to adjust the points again AFTER buffering, since buffering undoes the antimeridian adjustments
+      utility.adjustGeoCoords(bufferedLine)
+      const outlinePositions = bufferedLine.geometry.coordinates[0].map(
+        (coord) => Cesium.Cartesian3.fromDegrees(coord[0], coord[1], coord[2])
+      )
+      return new Cesium.PolylineGeometry({
+        positions: outlinePositions,
+        height: this.height,
+        width: this.width < 1 ? 1 : this.width,
+        vertexFormat: Cesium.EllipsoidSurfaceAppearance.VERTEX_FORMAT,
+        ellipsoid: this.ellipsoid,
+        followSurface: false,
       })
     }
     return _
   })()
+  const vertexSvg = `
```

Comment:
@andrewkfiedler I want these SVGs in separate files, but I wasn't sure how to do that with the way I'm loading them in below.